static/css/restyle.less

```diff
@@ -28,6 +29,17 @@
     src: local('Open Sans Italic'), local('OpenSans-Italic'), url('../../css/impala/fonts/OpenSans-Italic.ttf') format('ttf');
 }
 
+// Mixins
+.drop-shadow {
```

Comment:
I guess not, but I like the notion of unifying styles and using a constant mix-in. But I can take it out, given it’s not really re-used, fair enough.